lib/testmap.py

```diff
@@ -21,211 +21,215 @@
 
 from lib.constants import TEST_OS_DEFAULT
 
-COCKPIT_SCENARIOS = {'networking', 'storage', 'expensive', 'other'}
+COCKPIT_SCENARIOS = {"networking", "storage", "expensive", "other"}
 
 
 def contexts(image, *scenarios: Iterable[str], repo: Optional[str] = None):
-    return [image + '/' + '-'.join(i) + (('@' + repo) if repo else '')
-            for i in itertools.product(*scenarios)]
+    return [image + "/" + "-".join(i) + (("@" + repo) if repo else "") for i in itertools.product(*scenarios)]
 
 
 REPO_BRANCH_CONTEXT = {
-    'cockpit-project/bots': {
+    "cockpit-project/bots": {
         # currently no tests outside of GitHub actions, but declares primary branch
-        'main': [],
+        "main": [],
     },
-    'cockpit-project/cockpit': {
-        'main': [
-            *contexts('arch', COCKPIT_SCENARIOS),
-            *contexts('debian-stable', COCKPIT_SCENARIOS),
-            *contexts('debian-testing', COCKPIT_SCENARIOS),
-            *contexts('ubuntu-2204', COCKPIT_SCENARIOS),
-            *contexts('ubuntu-stable', COCKPIT_SCENARIOS),
-            *contexts('fedora-38', COCKPIT_SCENARIOS),
-            *contexts('fedora-39', COCKPIT_SCENARIOS),
+    "cockpit-project/cockpit": {
+        "main": [
+            *contexts("arch", COCKPIT_SCENARIOS),
+            *contexts("debian-stable", COCKPIT_SCENARIOS),
+            *contexts("debian-testing", COCKPIT_SCENARIOS),
+            *contexts("ubuntu-2204", COCKPIT_SCENARIOS),
+            *contexts("ubuntu-stable", COCKPIT_SCENARIOS),
+            *contexts("fedora-38", COCKPIT_SCENARIOS),
+            *contexts("fedora-39", COCKPIT_SCENARIOS),
             # this runs coverage, reports need the whole test suite
-            *contexts(TEST_OS_DEFAULT, ['devel']),
-            *contexts(TEST_OS_DEFAULT, ['firefox'], COCKPIT_SCENARIOS),
+            *contexts(TEST_OS_DEFAULT, ["devel"]),
+            *contexts(TEST_OS_DEFAULT, ["firefox"], COCKPIT_SCENARIOS),
             # no udisks on CoreOS → skip storage
-            *contexts('fedora-coreos', COCKPIT_SCENARIOS - {'storage'}),
-            *contexts('rhel-9-4', COCKPIT_SCENARIOS),
-            *contexts('rhel4edge', COCKPIT_SCENARIOS),
+            *contexts("fedora-coreos", COCKPIT_SCENARIOS - {"storage"}),
+            *contexts("rhel-9-4", COCKPIT_SCENARIOS),
+            *contexts("rhel4edge", COCKPIT_SCENARIOS),
+            *contexts("opensuse-tumbleweed", COCKPIT_SCENARIOS),
```

Comment:
Ah no, you added some changes. For these you first need to add them into `__manual` so they are not autotriggered when this PR would land. Then it is possible to run projects on these new images and if they work you can move them to default branches. Note that they won't work immediately as some tests definitely will need conditions (e.g. don't run rhel specific stuff, use different commands here and there...)